XDMA/linux-kernel/xdma/xdma_netdev.h

```diff
@@ -0,0 +1,95 @@
+#ifndef XDMA_NETDEV_H
+#define XDMA_NETDEV_H
+#endif
+
+#include <linux/netdevice.h>
+#include <linux/etherdevice.h>
+#include <linux/skbuff.h>
+#include <linux/workqueue.h>
+#include <linux/mutex.h>
+#include <linux/spinlock.h>
+
+#include "xdma_mod.h"
+
+#define RX_METADATA_SIZE 16
+#define TX_METADATA_SIZE 8
+
+#define DESC_REG_LO SGDMA_OFFSET_FROM_CHANNEL + 0x80
+#define DESC_REG_HI SGDMA_OFFSET_FROM_CHANNEL + 0x84
+
+#define DESC_REG_LO_RX SGDMA_OFFSET_FROM_CHANNEL_RX + 0x80
+#define DESC_REG_HI_RX SGDMA_OFFSET_FROM_CHANNEL_RX + 0x84
```

Comment:
Must use parenthesis when defining an expression
```suggestion
#define DESC_REG_LO (SGDMA_OFFSET_FROM_CHANNEL + 0x80)
#define DESC_REG_HI (SGDMA_OFFSET_FROM_CHANNEL + 0x84)

#define DESC_REG_LO_RX (SGDMA_OFFSET_FROM_CHANNEL_RX + 0x80)
#define DESC_REG_HI_RX (SGDMA_OFFSET_FROM_CHANNEL_RX + 0x84)
```